### ISSUE TITLE
Nyzul Isle - Frog & Custard boss silence audit

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#4978]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

I tried to look old videos for this but I could not find a single video where people silenced Custard/Frog bosses in nyzul (aside 1 whm silence resist on custard) or where a gekko/guillotine silence actually stuck and prevented casting. It would make sense they arent silancable since silencing more or less completely negates their kits and makes them pushovers. They are easily silencable on eden.

tldr: Could someone go to nyzul on retail and try to silence a frog, custard, soulflayer and/or an imp boss

**Comment by Steam-arch** [comment]
![image](https://user-images.githubusercontent.com/69089249/163592640-9c3a0cff-0162-4fb7-abb7-e9464aeb9f62.png)
Nevermind then! Maybe custards get a small buff when they get amorphic scythe/spikes.

Closing this.
